libs/foundry-dev-tools/src/foundry_dev_tools/clients/build2.py

```diff
@@ -191,3 +191,79 @@ def api_submit_build(
             json=body,
             **kwargs,
         )
+
+    def submit_dataset_build(
+        self,
+        dataset_rid: api_types.DatasetRid,
```

Comment:
@nicornk Do you think `dataset_rid` is fine or should it rather be a set of dataset rids to enable the option of submitting multiple dataset builds at once?